test/gtest/ucp/test_ucp_context.cc

```diff
@@ -79,13 +79,14 @@ UCS_TEST_P(test_ucp_version, wrong_api_version) {
     ucp_context_h ucph;
     ucs_status_t status;
     {
-        wrap_errors();
+        hide_warnings();
         status = ucp_init_version(99, 99, &params, config.get(), &ucph);
         restore_errors();
     }
-    if (status == UCS_OK) {
+    if (status != UCS_OK) {
+        ADD_FAILURE() << "Failed to create UCP with wrong version";
```

Comment:
need to also check that we had a warning (by a counter?)

Reply:
added warning count evaluation